src/js/getNextJokeIdx.spec.js

```diff
@@ -0,0 +1,15 @@
+import { getNextJokeIdx } from './getNextJokeIdx';
```

Comment:
You might want to default export the `getNextJokeIdx` function if it's the only function you plan on having in that file, then you don't have to use named imports.